activerecord/lib/active_record/connection_adapters/abstract/quoting.rb

```diff
@@ -73,12 +134,12 @@ def quote_string(s)
 
       # Quotes the column name. Defaults to no quoting.
       def quote_column_name(column_name)
-        column_name.to_s
+        self.class.quote_column_name(column_name)
       end
 
       # Quotes the table name. Defaults to column name quoting.
```

Comment:
This method does not call `quote_column_name` anymore, "Defaults to column name quoting." needs update.